### PR TITLE
Demo: ODE solver sensitivity

### PR DESCRIPTION
**Pull request by blide**

### CODE REVIEW

**Comment by vgvassilev** [comment]
@blide, can you run that demo adding a line in `test/Misc/RunDemos.C`

File: demos/ODESolverSensitivity.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+double solution(double a, double b, double c, double x) {
+  return rungeKutta(0, 0, x, 0.001, a, b, c);
+}
+
+// Evaluate b-sensitivity over x domain
+double bSensitivity(double x) {
+  // Using reverse mode since forward mode doesn't support
+  // differentiation of multi-arg calls
```
Comment (vgvassilev) [inline demos/ODESolverSensitivity.cpp:66]: Can you open an issue and reference it in a FIXME?

**Comment by blide** [comment]
Should I add checks for a generated code? I guess it's too long to make sense testing it.

**Comment by vgvassilev** [comment]
No, we want to make sure it runs successfully, no need to check generated code. How long does it run and can make it faster by reducing iterations?

**Comment by blide** [comment]
It runs in milliseconds.

**Comment by vgvassilev** [comment]
Then we are good. Let's add it there.

File: demos/ODESolverSensitivity.cpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  // FIXME: switch to clad::differentiate after fixing issue#168
```
Comment (vgvassilev) [inline demos/ODESolverSensitivity.cpp:67]:
```suggestion
  // FIXME: switch to clad::differentiate after fixing vgvassilev/clad#168
```

**Pull request merged**